mono/io-layer/events.c

```diff
@@ -89,35 +59,50 @@ static void event_ops_init (void)
 		(WapiHandleCapability)(WAPI_HANDLE_CAP_WAIT | WAPI_HANDLE_CAP_SIGNAL));
 }
 
-static void event_signal(gpointer handle)
+static const char* event_handle_type_to_string (WapiHandleType type)
 {
-	SetEvent(handle);
+	switch (type) {
+	case WAPI_HANDLE_EVENT: return "event";
+	case WAPI_HANDLE_NAMEDEVENT: return "named event";
+	default:
+		g_assert_not_reached ();
+	}
 }
 
-static gboolean event_own (gpointer handle)
+static gboolean __event_own (gpointer handle, WapiHandleType type)
 {
```

Comment:
We should use another naming convention instead of starting function names with __.